### ISSUE TITLE
Why the windows version was not merged to Electron?

### ISSUE THREAD

**Issue by wdanilo** [arkenthera/electron-vibrancy#33]
Hi! This is amazing. Lack of Windows support in Electron is so sad having this solution. Would you be so nice and tell more about why it was not merged / if there are any plans of doing it ?

**Comment by Toby56** [comment]
Probably because it's kindof a hack and has some issues.

**Comment by bennymeg** [comment]
It might be possible to implement it properly with [NodeRT](https://github.com/NodeRT/NodeRT) and this [blur code example](https://docs.microsoft.com/en-us/uwp/api/windows.ui.composition.compositioneffectbrush?view=winrt-19041) or this [acrylic](https://docs.microsoft.com/en-us/windows/uwp/design/style/acrylic)  [code example](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.acrylicbrush?view=winrt-19041) (win 10 only).

**Comment by ksambhi** [comment]
According to this https://docs.microsoft.com/en-us/windows/win32/apiindex/uwp-apis-callable-from-a-classic-desktop-app, APIs must have the DualApiPartition attribute to be called outside of UWP, which https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.acrylicbrush?view=winrt-19041 does not have.  It seems to call it you need a "package identity" (https://docs.microsoft.com/en-us/windows/win32/apiindex/uwp-apis-callable-from-a-classic-desktop-app).

This could be of use: https://blogs.windows.com/windowsdeveloper/2019/10/29/identity-registration-and-activation-of-non-packaged-win32-apps/

Also there's WinUI 3....

**Comment by bennymeg** [comment]
You are right, the `AcrylicBrush` do not has `DualApiPartition` attribute, but the `CompositionEffectBrush` (for blurring) has.

**Comment by danielehrhardt** [comment]
Did someone get it to work?

**Comment by danielehrhardt** [comment]
https://github.com/electron/electron/pull/7898

**Comment by drunkwinter** [comment]
Fixed in my fork along with a cleanup, package updates and improvements. https://github.com/DrunkWinter/electron-vibrancy
